src/ui/components/UserCollection/index.js

```diff
@@ -0,0 +1,72 @@
+/* @flow */
+import invariant from 'invariant';
+import * as React from 'react';
+import { compose } from 'redux';
+
+import translate from 'core/i18n/translate';
+import Link from 'amo/components/Link';
+import LoadingText from 'ui/components/LoadingText';
+import type { I18nType } from 'core/types/i18n';
+
+import './styles.scss';
+
+type Props = {|
+  authorUsername?: string,
+  id: number,
+  loading?: boolean,
+  name?: string,
+  numberOfAddons?: number,
+  slug?: string,
+|};
+
+type InjectedProps = {|
+  i18n: I18nType,
+|};
+
+type InternalProps = { ...Props, ...InjectedProps };
```

Comment:
So the pattern has slightly evolved: we don't need `InjectedProps` anymore, you can spread `Props` into `InternalProps`:

```js
type InternalProps = {|
  ...Props,
  i18n: I18nType,
|};
```